### PR TITLE
Markdownify

### PR DESCRIPTION
**Pull request by brenthuisman**
:)

### CODE REVIEW

File: doc/nml.md [hunk 1]
```diff
@@ -0,0 +1,965 @@
+# Introduction
```
Comment (thorstenhater) [inline doc/nml.md:1]: While I am fine with changing over README (the Org renderer on GH is not a nice as I would like), this file is more my personal notes than anything else. Orgmode holds a number of advantages there (for me at least), so I would prefer keeping that. If it ever turns into more that that, fine ;)

**Review by thorstenhater — CHANGES_REQUESTED** [review]

**Review by thorstenhater — COMMENTED** [review]
Tiny comment on the TLDR.

File: doc/nml.org [hunk 2]
```diff
@@ -17,6 +20,8 @@ support.
 
 * NeuroML2 and LEMS
 
+*TL;DR*: NeuroML is a dialect of the LEMS format. All NeuroML is LEMS.
```
[the remark this answers is not in the document]
Comment (brenthuisman) [reply doc/nml.org:23]: Is NeuroML2 a library though? Isn't that jNML, implementing the NML2 spec?

[the remark this answers is not in the document]
Comment (thorstenhater) [reply doc/nml.org:23]: jnml is an executable to interpret LEMS/NML2

[the remark this answers is not in the document]
Comment (brenthuisman) [reply doc/nml.org:23]: The schemata, as in the XSD files, is then 'the library'?

[the remark this answers is not in the document]
Comment (thorstenhater) [reply doc/nml.org:23]: No, the 'library' is whatever is in CoreTypes. XSD is syntax only in this analogy.

[the remark this answers is not in the document]
Comment (brenthuisman) [reply doc/nml.org:23]: Right, but where do I find the library in https://github.com/NeuroML/NeuroML2/ ? Only schema's as far as I can see.

So, it seems to me the word 'library' can be left out: LEMS is an XML dialect/language, and NeuroML2 defines itself in that language.

**Review by thorstenhater — APPROVED** [review]
👍

**Pull request merged**